src/entity.ts

```diff
@@ -580,6 +580,15 @@ export namespace entity {
 
     if (typeof value === 'number') {
       if (value % 1 === 0) {
```

Comment:
what is the purpose of `value % 1`, this seems like it could use a comment (even though it wasn't added by you).

Reply:
Done.

Would it be preferable to do 
```ts
if (Number.isInteger(value)) {
...
}
```
instead? Here the [method's](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) name speaks for itself.

Reply:
the method name is nice :+1:

Reply:
Swapped the logic to`Number.isInteger(value)`